pkg/plugin/twinmaker/client.go

```diff
@@ -428,15 +428,15 @@ func (c *twinMakerClient) GetPropertyValueHistory(ctx context.Context, query mod
 	}
 
 	if c := query.ComponentTypeId; c != "" {
-		if query.Properties == nil || len(query.Properties) < 1 {
+		if len(query.Properties) < 1 {
```

Comment:
Fixing these errors in CI:
<img width="908" alt="Screenshot 2025-04-18 at 19 42 47" src="https://github.com/user-attachments/assets/c1b4c1f6-292d-453b-89bb-8af1c48b80d2" />